core/src/main/java/org/infinispan/xsite/BackupReceiver.java

```diff
@@ -17,10 +20,51 @@
  */
 public interface BackupReceiver {
 
-   Cache getCache();
+   Cache<?, ?> getCache();
 
    CompletionStage<Void> handleRemoteCommand(VisitableCommand command, boolean preserveOrder);
 
+   /**
+    * Updates the key with the value from a remote site.
+    * <p>
+    * If a conflict happens, the update can be discarded.
```

Comment:
Nitpick but how about instead of saying "If a conflict happens", we say "If a conflict occurs". The former doesn't sound correct to me.